lib/App/Asciio/stripes/image_box.pm

```diff
@@ -0,0 +1,281 @@
+package App::Asciio::stripes::image_box ;
+use parent qw/App::Asciio::stripes::editable_box2/ ;
+
+use Gtk3 -init;
```

Comment:
not very ASCII to have images! nevertheless it may be usable. How is it converted to ASCII when exporting?